CMakeLists.txt

```diff
@@ -36,7 +36,32 @@ find_package(Threads REQUIRED)
 
 if (Qt5_POSITION_INDEPENDENT_CODE)
     set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-endif ()
+endif()
+
+## Third Party files
+set(THIRD_PARTY_COMMON
```

Comment:
All your PRs have this weird bunch of old code; I think you fork's master branch may be out-of-date